src/sql/workbench/browser/scriptingActions.ts

```diff
@@ -25,14 +25,14 @@ export class ScriptSelectAction extends Action {
 		super(id, label);
 	}
 
-	public run(actionContext: BaseActionContext): Promise<boolean> {
+	public async run(actionContext: BaseActionContext): Promise<boolean> {
 		return scriptSelect(
```

Comment:
This is a functional change. Not sure what the impact would be.

Reply:
From the code path it just ensures an extra check for connection.

Functionally I haven't noticed issues but I'm going to be doing a test pass of all of these code paths before merging it.